views/index.jsx

```diff
@@ -0,0 +1,143 @@
+const React = require('react')
+const Layout = require('./shared/layout')
+
+
+function IntroductionSection() {
+  return (
+    <section className="introduction-section">
+      <div className="hero-container">
+        <div className="hero-content">
+          <h1 className="hero__header">Powered by you</h1>
+          <p className="hero__text">See something broken online?<br />Tell the world and get help with Roar from More Human Internet.</p>
+          <a className="btn btn--download" rel="noopener noreferrer" href="https://morehumaninternet.org">Free Download</a>
+          <button className="btn btn--watch">Watch Video</button>
+        </div>
+        <img className="hero__gif" src="demo_video.gif" />
+      </div>
+    </section>
+  )
+}
+
+function Modal() {
+  return (
+    <section className="modal modal--hidden">
+      <div className="modal__content">
+        <div className="modal__video">
+          <iframe src="https://www.youtube.com/embed/QH2-TGUlwu4" width="560" height="315" frameborder="0"></iframe>
+        </div>
+      </div>
+    </section>
+  )
+}
+
+function Step(props) {
+  return (
+    <div className="step">
+      <div className="step-explanation">
+        {props.icon}
+        <h3>{props.title}</h3>
+      </div>
+      <p>{props.description}</p>
+    </div>
+  )
+}
+
+function StepsSection() {
+  return (
+    <section className="steps-section">
+      <h1 className="steps-title">Bring attention to problems on the web</h1>
+      <div className="steps">
+        <Step
+          title="1. Find an issue"
+          description="Search is broken? Link goes nowhere? Image won’t load? Let’s get this fixed!"
+          icon={
+            <svg viewBox="0 0 418 355">
+              <path d="M38 0C17.005 0 0 15.8862 0 35.5V248.5C0 268.114 17.005 284 38 284H171V319.5H133V355H285V319.5H247V284H380C400.995 284 418 268.114 418 248.5V35.5C418 15.8862 400.995 0 380 0H38ZM38 35.5H380V248.5H38V35.5ZM190 71V159.75H228V71H190ZM190 177.5V213H228V177.5H190Z" fill="#FB945B"/>
+            </svg>
+          }
+        />
+        <Step
+          title="2. Take a screenshot"
+          description="Open the Roar! extension to automatically capture a screenshot of the issue."
+          icon={
+            <svg viewBox="0 0 30 34">
```

Comment:
All these inline `SVG`s make it really hard to read the code. Is it possible to split them to separate files?